### DISCUSSION TITLE
Looking for feedback on metric for the universe, with time dilation but no expansion

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#178] [Cosmology Models]
Here it is:

$$ds^2 = -(1-\frac{rH}{c})^{-2} c^2 dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2$$

----
Edit: In a [comment below](https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7436594) by @HanDeBruijn a correction to the above is:

$$ds^2 = -(1-\frac{rH}{c})^{2} c^2 dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2$$

Which simplifies to:

$$ds^2 = -(c-Hr)^{2} dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2$$

----

The idea here is that time is stretched, but not space. It's basic spherical coordinates but the $dt$ has the coefficient: 

$$( 1-\frac{rH}{c})^{-2} $$

And:

$$(1-\frac{rH}{c}) = \frac{1}{1+z}$$

$H$ here is just Hubble's constant, $\frac{1}{14 \text{ billion years}}$. There isn't an evolving Hubble parameter, so the subscript 0 isn't needed. 

Someone asked in another discussion referencing Vavrycuk's paper, asking about why you couldn't just have a time dilation factor without a space expansion factor. Something like:

$$ds^2 = -A(t)^2 dt^2 + dx^2 + dy^2 + dz^2$$

The problem with that is the time coefficient is changing over the time dimension, which leads to a zero Ricci curvature tensor. This is taken to be Minkowski spacetime with a coordinate transformation, and no time dilation is possible there.

However, the new metric, the first equation, changes the time coefficient over the radius. And that avoids the problem. 

Imagine you have a 3D sphere, and every point in it has a clock. Instead of all the ticks clock at the same rate, the clock at the origin runs at "normal" speed, and all the others run slower depending on their distance from the origin.

At $r = c/H$ you get a cosmological horizon. 

Any thoughts? If this seems valid, I'm curious if anyone has any expertise calculating the temperature of that horizon.

**Comment by RedshiftDrift** [comment]
Einstein's theory describes physics "inside the universe" and cannot be extended to the *universe as a whole*.  We cannot even speak of the "universe as a whole" if the universe is not finite.

I believe it is a philosophical mistake to confound the container and the content - this is similar to the [fallacy of composition](https://en.wikipedia.org/wiki/Fallacy_of_composition).  Any attempt of describing the universe with an equation that describes its content is not valid (such as the Friedmann-Lemaître solution of the Einstein field equations).

**Reply by ExpEarth** [reply]
I think you're on the right track here, but like I mentioned in another thread, you could alternatively just say that it is the speed of light that is highest at the origin and lowest at the horizon.  That's because we can just define time as distance/_c_.  Then it would be okay to change the time term with radius.

>  I'm curious if anyone has any expertise calculating the temperature of that horizon.

I do.  It's 29 K there at the rim, but it would take me pages to explain why!

**Reply by mikehelland** [reply]
@RedshiftDrift , I think we are in complete agreement. 

In FLRW, the observable universe is an infinitely small piece that's essentially ignored.

My metric focuses 100% on the observable universe. r has to be less than c/H.

So FLRW is the universe beyond the observable. My metric is the opposite, just the observable.

**Reply by mikehelland** [reply]
> I do. It's 29 K there at the rim, but it would take me pages to explain why!

That's for the metric I proposed? 

Any pointers to get me in the right direction?

Obviously, I was hoping if we could show this horizon had a temp of 2.7 K then it would have a tidy explanation of the CMB.

**Reply by ExpEarth** [reply]
I'm not sure how you planned to calculate a temperature for the metric.  Actually, this is the first I have heard of a metric temperature.

But the key to the puzzle lies in a more general approach.  I take it that you will be assuming a tired light mechanism.  If so you need to be able to account for the linear relationship between the CMB temperature and distance: _T_(z) = To (1 + _z_).  Here To is the local CMB temperature.  This is hard to explain in a tired light model, because without the assumption of cosmological expansion the blackbody spectrum is not preserved.  This is one of the biggest strikes against TL models, though that is not always recognized (especially in ACG!).  If you simply draw the line back to the Hubble radius, you would find the CMB to be about 50 K there, but I have a set of calculations related to my own model suggesting it is only 29 K.  I'm not sure what the source of this discrepancy could be. 

My model involves other assumptions, including: (1) the universe is a black hole; (2) the Hubble redshift is caused by gravity (as per my optical gravity mechanism) and (3) there is a force which prevents collapse of the universe into a singularity, which I term the Hubble luminosity.  It might sound complicated but it's all pretty simple!

**Reply by mikehelland** [reply]
Interesting. 

By temperature from the metric, let me explain. 

de Sitter's original coordinates look somewhat like the metric I'm proposing.

$$ ds^2 = -\left(1-\frac{r^2}{\alpha^2}\right)dt^2 + \left(1-\frac{r^2}{\alpha^2}\right)^{-1}dr^2 + r^2 d\Omega_{2}^2. $$

https://physics.stackexchange.com/questions/93319/can-hubble-redshift-be-interpreted-as-time-dilation

For the time coefficient, there's something similar going on. But not quite. Specifically he has:

$$1 - \frac{r^2}{R^2} $$

And mine calls for:

$$(1- \frac{r}{R})^2$$

assuming R = c/H.

In this metric the horizons have, as I understand it, a temperature associated with them:

https://ned.ipac.caltech.edu/level5/Sept02/Padmanabhan/Pad10_3.html

https://physics.stackexchange.com/questions/92808/what-is-the-de-sitter-temperature-as-of-now

I was thinking that the horizons in my metric (r=c/H) should have a different temperature than de Sitter's, because the metric is different. So, maybe you can point me in the right direction.

For starters, this is not a tired light model. 

In this metric, light that is observed to redshifted and time dilated actually started that way. It didn't happen while traveling here.

Say something is happening 7 billion light years away, or 0.5c/H.

$$dt' = \frac{dt}{1-rH/c}$$
$$dt' = \frac{dt}{0.5}$$

In this metric, the time it takes to happen will 2x as long at that distance (0.5c/H, which is z=1). Light's frequency will be half, since f=1/t.

**Reply by budrap00** [reply]
Hi Mike,

You've got the right idea but you're still stuck with some residual concepts from the standard model. The metric you propose is a good one; it can be thought of as simply a polar framework that is locally appropriate to our unique Point of View of the Cosmos. This is a relativistic POV in which we are at the center of _our observable universe_. An observer on some distant galaxy would not see themself at the edge of a universe centered on us; they would see themself at the center of their own unique observable universe. Taken that way your framework is good, but if you take it to mean that distant galaxies are at the edge of the unitary Universe as conceived by the standard model you have made the FLRW mistake of assuming a "universal metric" with physically meaningful space and time coordinates.

From that assumption of a _universal metric_ flows all the conceptual errors that bedevil the standard model, but the situation can't be fixed by just assuming a different _universal metric_. It is an erroneous conceit that the Cosmos can be modelled with some simplistic mathematical framework describing a simultaneously existing Universe which does not in fact, at least by the known laws of physics, exist. In other words there ain't no such animal as the simultaneously existing Universe so any model that attempts to describe one will not resemble the Cosmos we actually observe.

The POV approach to understanding and modelling the Cosmos arises from the commonly understood behavior of electromagnetic radiation. Both a 2-Dimensional qualitative framework, light cones, and the 3-D version, expanding spherical wavefronts. have an observer-centered polar framework inherently present:

[Light Cones](https://thisislanduniverse.com/2023/04/05/light-cone-confusion-in-the-here-and-now/)
[Expanding Spherical Wavefronts](https://thisislanduniverse.com/2023/08/15/spherical-wavefronts-cosmological-reality/)

The Hubble parameter value you are using is an artefact of the standard model - 14 billion years has no significance outside that model. The value of the parameter indicates only the nature of the redshift-distance relationship which depends on the redshift mechanism and is therefore undetermined at this point. It has no real meaning as an expression of a model-dependent universal age. 

The CMB is present in the ESW analysis as it is in Olber's paradox; the night sky is bright - at 2.7K. Olber just didn't know about the cosmological redshift. Your polar framework should be adequate for calculating that value assuming a reasonable local-scale average galactic density value for the observed Cosmos.

**Reply by mikehelland** [reply]
@budrap00 Thanks for your response. I replied right below, in the next comment down.

https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7407616

**Reply by HanDeBruijn** [reply]
@mikehelland writes:
>I'm curious if anyone has any expertise calculating the temperature of that horizon.

@ExpEarth writes:
>I do. It's 29 K there at the rim, but it would take me pages to explain why!

I don't. I think it's $2.726$ K everywhere and it takes me [one page](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) to explain (apart from preliminaries).

>to account for the linear relationship between the CMB temperature and distance: T(z) = To (1 + z). Here To is the local CMB temperature.

Matt, may I ask where that _linear relationship_ does come from?

>My model involves other assumptions, including: (1) the universe is a black hole; (2) the Hubble redshift is caused by gravity (as per my optical gravity mechanism) and (3) there is a force which prevents collapse of the universe into a singularity, which I term the Hubble luminosity.

My model involves the same assumptions, with CMB instead of "the Hubble luminosity" aka redshift.
And the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) is taking care of singularities and other paradoxes.

**Reply by HanDeBruijn** [reply]
@mikehelland writes:
>In this metric the horizons have, as I understand it, a temperature associated with them:

The de Sitter temperature is $T=2.67\times10^{-30}\mbox{ K}$. Which is even much less than the miserable "temperatures" of the [Hawking radiation](https://en.wikipedia.org/wiki/Hawking_radiation) with Black Holes and the hypothetical [Unruh effect](https://en.wikipedia.org/wiki/Unruh_effect). Still apart from improper usage of the concept of Entropy here and everywhere.

**Reply by HanDeBruijn** [reply]
>Here it is:

$$
ds^2 = -(1-\frac{rH}{c})^{-2} c^2 dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2
$$

You have introduced a **singularity** for $r=c/H$, 14 billion years ago in time. Isn't that simply a **Big Bang** !?

**Comment by mikehelland** [comment]
> From that assumption of a _universal metric_ flows all the conceptual errors that bedevil the standard model, but the situation can't be fixed by just assuming a different _universal metric_. 

Understood. 

The metric intends to describe the past light cone of an observer, not the universe as a whole.

Perhaps that is not the end of the story though? 

The past light cone should be a sub-manifold within a manifold right?

I don't think the intention should be to describe the universe as a whole. But having a way to get from one light cone to another light cone (different frame of reference) would be a basic requirement I would think.


> The Hubble parameter value you are using is an artefact of the standard model - 14 billion years has no significance outside that model. The value of the parameter indicates only the nature of the redshift-distance relationship which depends on the redshift mechanism and is therefore undetermined at this point. 

I understand where you're coming. 

In this metric the redshift is caused by time dilation. A time dilated electromagnetic wave would have a longer period, and lower frequency and energy.

The significance of 14 billion years is probably a bit different in this metric than you'd think.

Let's say you setup a surveillance camera, and at Monday, at 00:001 (12:01 AM), you turned it on for the first time.

On Tuesday, at 00:01 AM, just 24 hours later, you go to check the tapes, and you have 32 hours of footage.

How is that possible?

There was a bug in the software. When you started the system, every frame it meant to record, it doubled. Over the course of the day the system recorded less and less doubles, and by noon, is was only doubling every other frame. At the end of the day, the problem corrected itself and only 1 frame was recorded per intended frame. 

Your buddy Joe says, "he, since you have 32 hours, go back to Sunday night, I want to see what happened."

And you try to explain, I have 32 hours of footage but it's really only 24 hours of time.

This is kind of like light coming to us from z=1. It started at z=1, but passed through z=0.9, z=0.8, z=0.7 and so on down to z=0. Every second of it's journey is time dilated by a different amount.

That's where the 14 billion years comes in.

Light from the edge of the observable will have a z=infinity and thus zero energy were the light to make it to us after an infinite time. But it would only have 14 billion years of "footage."

Observers of the universe 14 billion years from will see us smeared out to infinity too.

**Reply by HanDeBruijn** [reply]
@budrap00 writes:
>the FLRW mistake of assuming a "universal metric" with physically meaningful space and time coordinates.

Exactly: **metric is not physic** !!

@mikehelland :
I do not understand why you are stoking [another little campfire](https://github.com/orgs/a-cosmology-group/discussions/177#discussioncomment-7396061) that seems to replace your [Time Dilated Past hypothesis](https://github.com/orgs/a-cosmology-group/discussions/160).
In a Static Euclidean infinite and eternal Universe there are **many** timescales that show a [Time Dilated Past](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7217060):

$$
t=\frac{1}{H}\ln\left(\frac{1}{1+z}\right)=\frac{p}{H}\ln\left(\left[\frac{1}{1+z}\right]^{1/p}\right)=\frac{p}{H}\ln\left(H/p.\tau+1\right)
$$

$$
H/p.\tau+1=(1+z)^{-1/p} \quad\Longrightarrow\quad d\tau=\frac{p}{H}(-1/p)\frac{dz}{(1+z)^{1+1/p}}
$$

$$
dt=-\frac{1}{H}\frac{dz}{1+z} \quad\wedge\quad d\tau=-\frac{1}{H}\frac{dz}{(1+z)(1+z)^{1/p}} \quad\Longrightarrow\quad \large \boxed{dt=(1+z)^{1/p}.d\tau}
$$

The special cases $p=1$ and $p=2$ have been covered in that [previous post](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7217060).

**Reply by mikehelland** [reply]
This metric is for the time dilated past, not a replacement. 

Since:

$$\frac{1}{1+z} = e^{Ht} = 1 - \frac{rH}{c}$$

Integrating over them leads to the same results (in so far as the past light cone is concerned), but using a different manifold.

To answer your other question:

>14 billion years ago in time. Isn't that simply a Big Bang !?

Well, no. Because distances are static. 

14 billion years ago an observer will see a similar universe to the one we see today. They could peer back 14 billion years too (so 28 billion years back to us), but the last second would be infinitely time dilated; same as we see.

**Reply by HanDeBruijn** [reply]
This formula is from [another post](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7153006) of yours:

$$
d_T = \frac{c}{H_0} \log (1+z)
$$

With $d_T=$ _the light travel time distance_. It is supposed that $r=d_T$ . Then, isn't the above the same as

$$
r = \frac{c}{H}\ln(1+z) \quad \Longleftrightarrow \quad \frac{1}{1+z} = e^{Ht} = e^{-H r/c} \neq 1-\frac{rH}{c} = 1+H\tau \mbox{ (??)}
$$

As a first approximation it would be okay, but not for cosmological distances.

>using a different manifold.

Are you using Minkowski time instead of Measured time for re-defining _the light travel time distance_?

**Reply by ExpEarth** [reply]
@mikehelland 

> For starters, this is not a tired light model.  .... In this metric, light that is observed to redshifted and time dilated actually started that way. It didn't happen while traveling here.

Okay, it's not tired light, but then you still have to explain how the CMB is perfectly blackbody radiation at our position.  In the Big Bang model, this happens because they assume space is expanding.  The CMB energy density drops by a factor of (1 + z)^-3 relative to the situation if it's not expanding.  Without this the CMB cannot stay blackbody at each point.  For a long time it was just theoretical that the CMB temperature scales this way, but many observations have shown it to be the case:
[CMB temperatures.pdf](https://github.com/a-cosmology-group/acg/files/13198466/CMB.temperatures.pdf)
For a long time I disregarded this relationship, like others here in ACG, but in the end I found that it actually explains how gravity works (in my model).  So I am on board with it now.  I have been able to figure out how a tired light mechanism can keep the CMB as blackbody radiation at each point in space.  I can also adapt this to explain gravitational time dilation.  I'm not sure how you are covering this though.

On the other links you gave, I think those are false leads.  No one really has a good handle on black hole physics, so they make up a lot of stuff.

**Reply by HanDeBruijn** [reply]
@mikehelland .

The [Friedmann-Lemaître-Robertson-Walker metric](https://en.wikipedia.org/wiki/Friedmann%E2%80%93Lema%C3%AEtre%E2%80%93Robertson%E2%80%93Walker_metric) is defined as follows.

$$
ds^2 = c^2dt^2 - a(t)^2\left[\frac{dr^2}{1-kr^2}+r^2\left(d\theta^2+\sin^2(\theta)d\phi^2\right)\right]
$$

where $a(t)$ is known as the (cosmological) scale factor and $k$ is a constant representing the curvature of the space.
That so-called _curvature of space_ is of course **zero**, as it always has been: $k=0$.
This makes that the part between square brackets $\left[\mbox{ }\right]$ is simply the infinitesimal square length in Euclidean space.
Before the advent of Einstein, nobody had any trouble with such a (cartesian &rarr; spherical) coordinate transformation.
And I have never understood what is interesting about those metrics of General Relativity. But okay, if you want to adhere to it:

$$
a(t)=e^{Ht}=e^{-Hr/c}
$$

So, alternatively:

$$
ds^2 = - c^2\left(e^{Hr/c}\right)^2dt^2 + dr^2+r^2d\theta^2+r^2\sin^2(\theta)d\psi^2
$$

Which is **not** the metric as proposed by you, though approximately it is, for non-cosmological distances $r \ll c/H$ :

$$
\left(e^{Hr/c}\right)^2 = \left(e^{-rH/c}\right)^{-2} \approx ( 1-\frac{rH}{c})^{-2}
$$

**Reply by RedshiftDrift** [reply]
Interesting comment you make @HanDeBruijn 
> I have never understood what is interesting about those metrics of General Relativity

Indeed, mathematically those metrics are simple coordinate transformations.

**In Euclidean space** the laws of physics apply without any "correction".  When a coordinate transformation is applied then this gives rise to all sorts of "fictitious effects" that are not physical, such as a non-linear motion of the $\theta$-coordinate of an inertial object. (this is a simplified example...)

**In Relativistic space** defined by some coordinate transformation given by the metric, the laws of physics are valid ***in that coordinate system***.  If you convert back to Euclidean space then all sorts of "fictitious effects" appear.

So indeed, "before the advent of Einstein, nobody had any trouble with such a (cartesian → spherical) coordinate transformation." If you only look at the mathematics you don't understand what is interesting in those metrics.
But beyond the mathematics is the physical interpretation.  Of the two coordinate systems only one has a physical meaning: the *physics* happens in only one metric, the other one is a mathematical transformation.  What's interesting is which metric you choose as being the one with physical reality.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .
>What's interesting is which metric you choose as being the one with physical reality.

Very much agreed !! This is more or less similar to my findings in a [previous posting](https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7411544).
To my surprise, there are **infinitely many** time scales that lead to the well-known distance formula $r=c/H\ln(1+z)$.
Therefore it's impossible to decide - based upon that formula - which _Time Dilated Past_ would be the physical relevant one. Mike's favorite is $dt=d\tau(1+z)$ ; my favorite is $dt=dT\sqrt{1+z}$ ; other favorites are thinkable.

### Update
A nice example of a wrong reference frame in everyday physics is the following problem: [Strapping down a cylinder](https://math.stackexchange.com/questions/4774185/strapping-down-a-cylinder/).
The author of the [accepted answer](https://math.stackexchange.com/questions/4774185/strapping-down-a-cylinder/4774346#4774346) says that the length of the string is

$$
l(\alpha)=2\sqrt{h^2+r^2\alpha^2}+2r\cos\alpha
$$

with $r=$ radius, $h=$ height, $\alpha=$ angle. Crucial is the remark that $r\alpha$ comes from coordinates _on the unfolded cylinder_.
But the cylinder is **not** unfolded in todays physical reality.
As [my comment](https://math.stackexchange.com/questions/4774185/strapping-down-a-cylinder/4774346#comment10149262_4774346) says: _Almost, but not quite. Pythagoras is not for triangles with an arc in it. We must have instead_

$$
l(\alpha)=2\sqrt{h^2+4r^2\sin^2(\alpha/2)}+2r\cos\alpha
$$

For small angles $\alpha$ the lengths $l$ are indistinguishable.

**Reply by budrap00** [reply]
`...it's impossible to decide - based upon that formula - which Time Dilated Past would be the physical relevant one.
`

That is just a reflection of the straightforward fact that there is no scientific basis for the idea of a universal past anymore than there is for a universal present or a universal future. The absence of a universal space-time frame is a characteristic of Relativity Theory that is well supported by the known finite light speed limitation. The imaginary unitary Universe assumption of the FLRW model flies in the face of known physics and is the fundamental flaw of modern cosmology.

**Reply by HanDeBruijn** [reply]
@budrap00 .
>there is no scientific basis for the idea of a universal past anymore than there is for a universal present or a universal future.

That's right, but Time Dilated Past by Mike and myself is **not** a reflection, at all, of that "straightforward fact" of yours, as is clearly worded by Mike in a [previous post](https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7407616), notably meant as a reply to you:
>Observers of the universe 14 billion years from will see us smeared out to infinity too.

There is **nothing** claimed to be "universal" in our ideas about past, present or future.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .
I would like to have a second opinion about **table 1** in [CMB.temperatures.pdf](https://github.com/a-cosmology-group/acg/files/13198466/CMB.temperatures.pdf).
Question is: how can they "measure" CMB temperatures with such redshifts ?

**Reply by RedshiftDrift** [reply]
> how can they "measure" CMB temperatures with such redshifts ?

They actually measure the temperature of the gas in galaxies at such redshifts, and assume the gas is at equilibrium with the CMB.  The temperature of a gas is obtained from measurements of absorption-intensity ratios of molecular lines, which in principle is a good probe of the temperature.

However, the environment of the gas also has to be modeled, especially how much UV is present.  This cannot be modeled easily and astronomers add assumptions, some of which depend on the redshift.

So the temperatures listed in that table are based on the evolution of the universe according to the Big Bang model, and not surprisingly they get results in agreement with the Big Bang model.  It's just a vicious circle, the results are worthless.

See discussion [https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6775390](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6775390) and an older post from 2023 Jan 17 that I copy here, showing how the mistake is made for a specific example.

> From: Louis Marmet via A Cosmology Group a.cosmology+redshift_at_cosmology.info @ gaggle.email
> Sent: January 17, 2023 9:46 PM
> Subject: Re: [A Cosmology Group] CMB T at high z
>  
> 
> Hi Matt,
>   the papers masuring the temperature of the CMB at high-z assume the Big Bang model to analyze the data.  The fallacy of begging the question...
> 
>   I examined the paper by Kimenko we discussed on 2022-12-8 [[The measurements of the CMB temperature in diffuse interstellar medium...](https://dx.doi.org/10.1088/1742-6596/1697/1/012013)]  They "estimate the CMB temperature to 14.2 K at z = 3.287, that is in agreement with the expected TCMB(zabs) = 11.7 K."
> 
>   The flaw in their analysis is revealed in the caption of Figure 1:
> "analysis of excitation of H2 rotational and Ci fine-structure levels in the H2 absorption systems at z = 3.287 towards QSO J 0816+1446.   ... constraints on the number density and UV intensity obtained with the analysis of excitation of fine-structure levels of C i and lower rotational levels of H2 (J=0 to J=2), respectively. Other parameters of the fit were fixed to TCMB = 11.6 K and Tkin = 110 K..." (my highlight)
> 
>   In other words, H2 absorption lines give T_CMB = 14.2 K based on the calculation of various properties in the cloud (UV intensity, hydrogen density, kinetic temperature) _and_ the CMB temperature at z = 3.287, which they calculate to be T_CMB = 11.6 K using the Big Bang model!
>   I can't tell you how much an influence this has on the final result, but it certainly does not inspire confidence.
> 
> 
>   The paper ApJ, 934:64 (17pp), 2022 July 20 cited below by Eric reports measurements of the dust temperature in a galaxy at z = 7.13.  There is the problem mentioned by Eric, we are selecting brighter and brighter galaxies, and there is also a problem with the calculation of the star forming rate (SFR) in the galaxy.  The calculation of the SFR depends on correlations observed between emission lines, type of star populations, and the luminosity of the galaxy.  However the luminosity is incorrectly calculated in the Big Bang model.  This gives an incorrect estimate of star populations, skews the SFR, and changes the estimated temperature of the dust.  Again, the temperature of the dust is calculated from an analysis that is biased by the Big Bang model.
>   I can't tell you how much an influence this has on the final result, but it certainly does not inspire confidence in this case either.
> 
> 
>   Because of that, we may not have observations of dust temperatures that are less than the predicted CMB at high-z.  However, it's not because the temperatures are hotter but because the analyses are biased by the Big Bang assumptions and do not give us the real temperatures.
>

**Reply by ExpEarth** [reply]
@RedshiftDrift 

Thanks, Louis, for gathering these arguments together.  There are actually two main systems of temperature measurement, the one you describe with populating the electronic levels of gases, and one using the Sunyaev-Zel'dovich effect (SZ effect).  You are an expert on the SZ effect and so I won't try to describe that one to you.

You are saying in general that Big Bang assumptions are skewing the data.  You mention the caption to Fig. 2 (of the Klimenko et al paper), saying that by using the predicted CMB temperature (in the Big Bang model) to model other aspects of the cloud they are producing a worthless result.  I don't think that is a fair statement without first wading through all their analysis and data and seeing what the difference really is in making the assumption they did.  It could be just a minor effect.  Basically, you are saying that in two types of measurement with multiple papers for each that Big Bang assumptions are clouding the results.  I can't say whether this is right or wrong.  I tried myself to uncover the flaws in the CO study and couldn't find it.  The ACG readers could happily go on neglecting these studies and I could not blame them.  It is my own model which leads me to believe that these studies are extremely likely to be correct.  There was a more recent study involving water molecules at z = 6.34 which show the same result as well.

@mikehelland  Lastly, I did not intend to deflect attention from your question.  I was just curious as to how you saw this aspect.

**Reply by budrap00** [reply]
@HanDeBruijn The problem is that you and Mike say things like:
 
`There is nothing claimed to be "universal" in our ideas about past, present or future.`

But you then turn around and say things like this:

`Observers of the universe 14 billion years from (now) will see us smeared out to infinity too.`

I added _now_ to that statement assuming its absence a typo. In that context the term implies a _universal now_. You are speaking of the Cosmos as if it were a simultaneously existing entity - what modern cosmology calls the Universe. Even then the statement, on its face, doesn't make any sense since it doesn't specify which "observers of the universe", located where, it applies to. 

It certainly can't apply to _all_ possible observers. It can't apply to observers who might exist 14Gy in the future of the Milky Way or of a nearby galaxy like Andromeda. The statement asserts knowledge of observers at some three dimensional locations that we cannot observe now let alone at some time 14Gy in the future of some unspecified time frame. I assume the value 14Gy derives from some math that shares a measure of commonalty with the standard model otherwise its a very peculiar coincidence. 

The bottom line for me is that quite apart from whatever your models' mathematical merits are, they are steeped in a qualitative framework of "universal" concepts. I don't think this is just a matter of semantics either; it's not simply that you employ the ambiguous term universe with its inherent "unitary" baggage.. 

You explicitly seem to agree with the arguments against a universal framework but that does not prevent you from employing such a framework in the way you speak (and I assume therefore think) about the Cosmos. In other words your explicit understanding is at odds with the implicit qualitative framing of your models. Your models will make more sense if they are qualitatively framed in a way that is consistent with the inherent observational constraints of physical reality. If your math stretches beyond those constraints to encompass a "universe" then you have an imaginary model of the Cosmos that will ultimately be as scientifically meaningless as a mathematical model of the StarTrekWars universe(s).

No 3D localized observers like ourselves can observe "the universe" as a simultaneous entity, so statements about "observers of the universe" don't make scientifically logical sense. Its a kind of vague metaphysics without any clear scientifically definable meaning because it refers to an entity, "the universe", that does not exist in any meaningful scientific way.

**Reply by mikehelland** [reply]
I think you make a fair point.

Speaking of the universe is a bit of habit, and basically unavoidable when engaging with the mainstream models. 

It should be pointed out that my metric does not describe a homogeneous universe like FLRW does.

I think the strategy I've sort of unwittingly happened on is to suppose that the past light cone can be a manifold, and my metric describes the coordinates to that.

So to clarify and extend my statement: an observer existing 1 trillion in the past, or 1 trillion years in the future, should have similarly shaped past light cones as we do.

**Reply by OlliSant** [reply]
We can extrapolate in time and space, and that all is the universe. It is an exact entity in the reality. It is the universe. It is different as the observable universe. We can make a theory of it and test it in astronomy. And form it in philosophy and astronomy and cosmology. Together all these sciences. Testing needs astronomical scills.

**Reply by OlliSant** [reply]
GR changed to 4D and Q and no beginning, eternal time. No expansion. 

 Where all four dimensions are similar vectors, space dimensions, ballike structure, Q is diameter, the longuest possible distance in reality. Matter is galaxies, Galaxies,  Explosions, quasars, smokes and forces and fields and radiations and waves. 

Must be tested in astronomy and made better 60 years.

This is the whole universe, and the local universe is about BB.

**Comment by OlliSant** [comment]
Model for the whole universe, no expansion, no outside but all space, very big, boundless, eternal time, no beginning, 4D. Time not a dimension. What else is needed? Ball like, diameter the longest possible distance. How is that? How to say that in mathematics?

**Comment by bgreenman** [comment]
I support the replies written 2 days ago by redshiftdrift and budrap00, observers 14 billion light years away would see the same view as we see. Gravity could be used a model for the CMB origins. mikehelland you are making a good try to explain non-standard cosmology, keep it up. ollisant is trying to say it in mathematics

**Comment by mikehelland** [comment]
@HanDeBruijn 

> With dT= _the light travel time distance_. It is supposed that r=dT . 

Not quite.

Light travel time distance is $-ct$, the speed of light times the measured (time dilated) time, $t$.

The $r$ coordinate is shorter, it is the speed of light times the Minkowski (un-time dilated) time, $\tau$.

$$  r = -c\tau = \frac{c}{H} \frac{z}{1+z}  = -\frac{c}{H} (e^{Ht} - 1) $$

Since we're talking about the past $\tau$ and $t$ should be negative ($t=\tau=0$ is the present). The absolute value of $t$ should be greater than the absolute value of $\tau$ because $t$ is a time dilated $\tau$.

So:

$$ \frac{1}{1+z} = e^{Ht} = 1 - \frac{Hr}{c} $$

**Reply by bgreenman** [reply]
correct

**Reply by HanDeBruijn** [reply]
@mikehelland .

Ah, now we are getting somewhere .. You started with

$$
ds^2 = -(1-\frac{rH}{c})^{-2} c^2 dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2
$$

I have found the singularity suspect from the beginning, but okay, that's just a feeling ..
Then you write

$$
\frac{1}{1+z} = e^{Ht} = 1 - \frac{Hr}{c}
$$

We [also have](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7217060)

$$
dt = (1+z).d\tau \quad \Longrightarrow \quad d\tau^2 = (1+z)^{-2}dt^2 = (1-\frac{Hr}{c})^2 dt^2
$$

But $\tau$ is Minkowski time, so Special Relativity (flat space-time) should be the result:

$$
ds^2 = -c^2 d\tau^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2
$$

Your metric is upside down:

$$
ds^2 = -(1-\frac{Hr}{c})^2 c^2 dt^2 + dr^2 + r^2 d\theta^2 + r^2\sin^2 (\theta) d\psi^2
$$

Singularity removed. And .. **contradiction** !!

**Reply by mikehelland** [reply]
Hmmm. Could it be simplified further?

If:

$$ 1 - \frac{Hr}{c} = \frac{c - Hr}{c} $$

Then:

$$\left(  \frac{c - Hr}{c} \right)^2 c^2 = (c - Hr)^2 $$

**Reply by bgreenman** [reply]
Keep working on correcting the equations

**Reply by HanDeBruijn** [reply]
So far so good. But this is not the end of the story.
With the Variable Mass Theory "instead" of Time Dilated Past we have, analogously:

$$
\frac{1}{1+z} = e^{Ht} = \left(1+\frac{H}{2}T\right)^2
$$

$$
dt = dT\sqrt{1+z} \quad \Longrightarrow \quad dT^2 = (1+z)^{-1}dt^2 = \left(1+\frac{H}{2}T\right)^2 dt^2
$$

But, according to [Arp and Narlikar](http://www.alternatievewiskunde.nl/cosmology/narlikar.pdf), with timescale $T$, flat (Minkowskian) space-time should be the result:

$$
ds^2 = d\Sigma^2 - c^2 dT^2 \quad \mbox{with} \quad d\Sigma^2 = dx^2+dy^2+dz^2 = dr^2+r^2d\theta^2+r^2\sin^2(\theta)d\psi^2
$$

And so

$$
ds^2 = d\Sigma^2 - c^2\left(1+\frac{H}{2}T\right)^2dt^2
$$

I hesitate to replace $T$ by $(-r/c)$ because I'm not sure **it's the same** $r$ **(??)** as in the $\Sigma$ (space) part of the metric.
Same problem for you @mikehelland ??

**Reply by bgreenman** [reply]
A correct response

**Reply by mikehelland** [reply]
@HanDeBruijn 

I'm not familiar with that. 

~~I don't think $T$ can be in your metric tensor if it's not one of the coordinates.~~

I see, T is a constant. I'll have to get deeper into that paper.

**Reply by HanDeBruijn** [reply]
>Same problem for you @mikehelland ??

$$
\frac{1}{1+z} = e^{Ht} = 1+H\tau
$$

$$
dt = d\tau(1+z) \quad \Longrightarrow \quad d\tau^2 = (1+z)^{-2}dt^2 = \left(1+H\tau\right)^2 dt^2
$$

$$
ds^2 = d\Sigma^2 - c^2\left(1+H\tau\right)^2dt^2
$$

Is it safe, Mike, to replace $\tau$ by $(-r/c)$ ? Because maybe **it's not the same** $r$ as in the $\Sigma$ (space) part of your metric:

$$
ds^2 = d\Sigma^2 - c^2 d\tau^2 \quad \mbox{with} \quad d\Sigma^2 = dr^2+r^2d\theta^2+r^2\sin^2(\theta)d\psi^2
$$

**Reply by mikehelland** [reply]
I think it should be the same because you're doing $(x_0 - x_1)^2 + (y_0 - y_1)^2 + (z_0 - z_1)^2$ where $x_0 = y_0 = z_0 = 0$.

Since $r = -c\tau$ then $\tau = -r/c$.

**Comment by bgreenman** [comment]
Its not easy to formulate a new model of a non-FLRW universe, i think it will eventually happen which is good. Developing, testing and verifying a new model takes considerable time.
